addons/web/static/src/webclient/settings_form_view/settings_form_controller.js

```diff
@@ -17,14 +17,19 @@ export class SettingsFormController extends formView.Controller {
     setup() {
         super.setup();
         useAutofocus();
-        this.state = useState({ displayNoContent: false });
-        // Deprecated warning: a new way to point to sections or items will be
-        // developed so that putting a default search value won't be necessary
-        if ("default_search_setting" in this.props.context){
```

Comment:
`default_search_setting` is currently used in : https://github.com/odoo/odoo/blob/25cbe460b56ed4cef956b6d7174534dfb5a67dfd/addons/l10n_it_edi/models/res_company.py#L147

Correct me if I'm wrong, but this will not work any more after this commit.

Reply:
Ah indeed 🙈 , I will change that as well.

Thanks! 🙂

Reply:
@jpp-odoo 
Actually, I cannot use my solution in their code. I guess I will have to re-introduce the previous solution. 😅

And I didn't use that solution because we cannot really translate a string in a real action defined in XML. 😕

Reply:
I wasn't very enthusiast of their solution, or this solution, I'm nota big fan of using the context to modify and execute the search.

We were thinking of adding a new way to scroll to a specific position on a view on the webclient, based on the fragment on the url, the development of this will follow up the merge of the PR: https://github.com/odoo/odoo/pull/142007

This means that you could only do : `href="/odoo/settings#my_id"` and it should open the settings view, and scroll to the specific setting.

Do you think that something like that, could solve your issue ?

Reply:
@jpp-odoo 

> This means that you could only do : href="/odoo/settings#my_id" and it should open the settings view, and scroll to the specific setting.

I don't know if it will work if the element with `my_id` is in another setting menu.

For instance, in accounting, their element to trigger is in `Invoicing` setting menu and not the `General settings` one.

If you think it will work in any settings menu then LGTM.

Reply:
> If you think it will work in any settings menu then LGTM.

The idea was to be able to work in all the views, maybe we will need to adapt a little bit the settings controller (as we did on notebook for this to work).